packages/frontend/src/components/common/modal/Style.css.js

```diff
@@ -63,7 +64,7 @@ const Style = styled.div`
         z-index: 1050;
         width: 100%;
         background-color: white;
-        border-radius: 6px;
+        border-radius: 16px;
```

Comment:
Not a dramatic change but this change would affect other modal component's border radius. Have you checked on other modal UIs?

Reply:
Yep I changed it globally since the Figma design is using 16px. In general we should make these changes globally to keep things consistent. UI design is constantly evolving so whenever we notice an inconstancy between a new design and the current UI let's confirm the change and then make it global, especially for common elements like modal, buttons, etc.